core/src/identity/rsa.rs

```diff
@@ -30,7 +30,7 @@ use std::{fmt, sync::Arc};
 use zeroize::Zeroize;
 
 /// An RSA keypair.
-#[derive(Clone)]
+#[derive(Debug, Clone)]
```

Comment:
With my suggestion above, we would not need this `Debug`.

Reply:
A good practice is to have `Debug` on all public data structures (pedantic preset in clippy suggests this IIRC).

Turns out other keypairs (ed25519 and secp256k1) already implemented `Debug` by printing public key, doing the same here for symmetry.